addon/motions/resize.js

```diff
@@ -35,30 +37,81 @@ export class Resize extends Motion {
     this.prior = motions.find(m => m instanceof this.constructor);
   }
 
+  /**
+   * Extracts initial/final height and width from the `this.opts` object. Required for sprites wrapped in a resize
+   * `animated-if`, as they may not have `initialBounds` or `finalBounds` of their own
+   * @private
+   */
+  _extractHeightAndWidthOpts() {
+    const returnVariables = {};
+    HEIGHT_AND_WIDTH_OPTS.forEach(optionKey => {
+      let bounds = 'initialBounds';
+      let transform = 'initialCumulativeTransform';
+      let transformProp = 'a';
+      let dimension = 'width';
+      let option = this.opts[optionKey];
+
+      if (optionKey.indexOf('to') > -1) {
+        bounds = 'finalBounds';
+        transform = 'finalCumulativeTransform';
+      }
+      if (optionKey.indexOf('Height') > -1) {
+        dimension = 'height';
+        transformProp = 'd';
+      }
+
+      if (option != null) {
+        returnVariables[optionKey] = option;
+        returnVariables[transform] = { ...this.sprite[transform], ...returnVariables[transform], [transformProp]: 1 };
+      } else {
+        let spriteBounds = this.sprite[bounds];
+        returnVariables[optionKey] = spriteBounds != null && spriteBounds[dimension];
+        returnVariables[transform] = { ...this.sprite[transform], ...returnVariables[transform] };
+      }
+    });
+
+    return returnVariables;
+  }
+
+
+
+
   * animate() {
     let sprite = this.sprite;
     let duration = this.duration;
+    let { fromHeight, toHeight, fromWidth, toWidth, initialCumulativeTransform,
+      finalCumulativeTransform } = this._extractHeightAndWidthOpts();
 
     if (!this.prior) {
-      this.widthTween = new Tween(
-        sprite.initialBounds.width / sprite.initialCumulativeTransform.a,
-        sprite.finalBounds.width / sprite.finalCumulativeTransform.a, duration
-      );
-      this.heightTween = new Tween(
-        sprite.initialBounds.height / sprite.initialCumulativeTransform.d,
-        sprite.finalBounds.height / sprite.finalCumulativeTransform.d, duration
-      );
+      this.widthTween = (fromWidth === false || toWidth === false)
+        ? { done: true }
```

Comment:
I think this will break when one resize motion interrupts another, because of this line below:

```js
).plus(this.prior.widthTween);
```

Instead of having a special case here for a missing `toWidth` or `fromWidth`, it may be easier to change `_extractHeightAndWidthOpts` so that if one of those is missing, it substitutes the other. So instead of returning

```js
{
  fromWidth: 100,
  toWidth: false,
  ...
}
```

it would become:

```js
  fromWidth: 100,
  toWidth: 100, // this automatically defaulted to being the same as fromWidth
```

This will give you a Tween that doesn't actually move, but that's what we want.